packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts

```diff
@@ -138,8 +139,18 @@ function getTransformerFactory(context, resourceDir, authConfig?) {
 
     // TODO: Build dependency mechanism into transformers. Auth runs last
     // so any resolvers that need to be protected will already be created.
-    transformerList.push(new ModelAuthTransformer({ authConfig }));
 
+    let amplifyAdminEnabled: Boolean = false;
+
+    try {
+      const amplifyMeta = stateManager.getMeta();
+      const appId = amplifyMeta.providers[providerName].AmplifyAppId;
+      amplifyAdminEnabled = await isAmplifyAdminApp(appId);
+    } catch (err) {
+      console.info('App not deployed yet.');
```

Comment:
Why is this try catch needed? Error is caught, but `info` is printed, then code flow continues..this logic needs to be cleaned up I think.

Reply:
This is for pre-deployment codegen since  amplifyMeta.providers[providerName].AmplifyAppId is not present, also const amplifyMeta = stateManager.getMeta(); fails pre-deployment.